{"schemaVersion":{"minor":3,"major":0,"patch":0},"metadata":{"externalID":"c:@M@MobilePaymentsSDKAPI@E@SQMPPaymentError@SQMPPaymentErrorUnexpected","symbolKind":"case","roleHeading":"Case","modules":[{"name":"MobilePaymentsSDK"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unexpected"}],"role":"symbol","title":"PaymentError.unexpected"},"sections":[],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI","doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/PaymentError"]]},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unexpected","kind":"identifier"}]}],"kind":"declarations"}],"variants":[{"paths":["\/documentation\/mobilepaymentssdkapi\/paymenterror\/unexpected"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"type":"codeVoice","code":"PaymentManager.startPayment"},{"type":"text","text":" was used in an unexpected or unsupported way. Check your local logs and try the payment again."}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/PaymentError\/unexpected"},"references":{"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI/PaymentError":{"kind":"symbol","type":"topic","url":"\/documentation\/mobilepaymentssdkapi\/paymenterror","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"PaymentError"}],"title":"PaymentError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"PaymentError"}],"identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/PaymentError","abstract":[{"text":"Errors relating to a payment failing.","type":"text"}]},"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI/PaymentError/unexpected":{"url":"\/documentation\/mobilepaymentssdkapi\/paymenterror\/unexpected","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unexpected"}],"type":"topic","abstract":[{"type":"codeVoice","code":"PaymentManager.startPayment"},{"type":"text","text":" was used in an unexpected or unsupported way. Check your local logs and try the payment again."}],"role":"symbol","title":"PaymentError.unexpected","kind":"symbol","identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/PaymentError\/unexpected"},"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI":{"abstract":[],"url":"\/documentation\/mobilepaymentssdkapi","title":"MobilePaymentsSDK","type":"topic","role":"collection","kind":"symbol","identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI"}}}